sqlengine/postgres_engine.go

```diff
@@ -70,14 +70,32 @@ func (d *PostgresEngine) Close() {
 	}
 }
 
-func (d *PostgresEngine) execCreateUser(tx *sql.Tx, bindingID, dbname string) (username, password string, err error) {
-	groupname := d.generatePostgresGroup(dbname)
-
-	if err = d.ensureGroup(tx, dbname, groupname); err != nil {
+func (d *PostgresEngine) execCreateUser(tx *sql.Tx, bindingID, dbname, masterUsername string, readOnly *bool) (username, password string, err error) {
+	if readOnly != nil {
+		d.logger.Debug("exec-create-user",
+			lager.Data{
+				"BindingID":      bindingID,
+				"DatabaseName":   dbname,
+				"MasterUsername": masterUsername,
+				"READ-ONLY":      *readOnly,
+			})
+	} else {
+		d.logger.Debug("exec-create-user",
+			lager.Data{
+				"BindingID":      bindingID,
+				"DatabaseName":   dbname,
+				"MasterUsername": masterUsername,
+				"READ-ONLY":      "nil",
+			})
+	}
```

Comment:
nit: the `if` block appears unnecessary here, lager.Data will happily accept a pointer here and do a sensible thing. consider simplifying the code by removing the if

nit: the casing of "READ-ONLY" looks bit odd to my eyes, maybe consider keeping it consistent with the others

```suggestion
		d.logger.Debug("exec-create-user",
			lager.Data{
				"BindingID":      bindingID,
				"DatabaseName":   dbname,
				"MasterUsername": masterUsername,
				"ReadOnly":      readOnly,
			})
```